media-gfx/blender/blender-2.79b.recipe

```diff
@@ -243,17 +242,16 @@ BUILD()
 		-DWITH_OPENMP=ON \
 		-DWITH_OPENVDB=ON \
 		-DWITH_OPENVDB_BLOSC=ON \
-		-DWITH_PYTHON_INSTALL=ON \
 		-DWITH_RAYOPTIMIZATION=ON \
 		-DWITH_SDL=ON \
 		-DWITH_PLAYER=OFF \
 		-DWITH_OPENSUBDIV=ON \
-		-DWITH_PYTHON_INSTALL=ON \
-		-DWITH_PYTHON_INSTALL_NUMPY=ON \
-		-DWITH_PYTHON_INSTALL_REQUESTS=ON"
+		-DWITH_PYTHON_INSTALL=OFF \
```

Comment:
This is also diverges from the release spec: https://github.com/blender/blender/blob/master/build_files/cmake/config/blender_release.cmake#L46

Reply:
This disables bundled python. Python on 32 bit Haiku is quite messed because of secondary architecture and bundled python is not working on 32 bit. It seems not generated correctly anyway even on 64 bit.

Reply:
Yep, but it means in each case the python recipe gots bumped blender have to be rebuilt i fear.

Reply:
Blender 2.79 is locked to python 3.6, so python will unlikely frequently update. Installing python on 32 bit will probably need adding x86 directories and changing install logic.